book/03-git-branching/sections/rebasing.asc

```diff
@@ -208,7 +208,7 @@ Or you could do it manually with a `git fetch` followed by a `git rebase teamone
 
 If you are using `git pull` and want to make `--rebase` the default, you can set the `pull.rebase` config value with something like `git config --global pull.rebase true`.
 
-If you treat rebasing as a way to clean up and work with commits before you push them, and if you only rebase commits that have never been available publicly, then you'll be fine.
+If you treat rebasing as a way to clean up and work with commits before anyone could have based work on them, either because they have not been available publicly or it is a branch that you are the only one supposed to work on, then you'll be fine.
```

Comment:
What about something more like this:

> If you only ever rebase commits that have never left your own computer, you'll be just fine.
>
> If you rebase commits that have been pushed, but that no one else has based commits from, you'll also be fine.

Reply:
Sounds good to me, it clearly cover the two options, rebase of local commits and rebase of commits that you are the only one working on them.